src/main.c

```diff
@@ -69,18 +69,23 @@ ux_state_t G_ux;
 #define OFFSET_CDATA 5
 
 static const uint8_t TOKEN_SIGNATURE_PUBLIC_KEY[] = {
-// cLabs production key (2020-04-14)
+  // cLabs production key (created: May 30, 2024, 17:18:49 GMT+2)
+  // akeyless URL: https://ui.gateway.akeyless.celo-networks-dev.org/items?id=281699245&name=%2Fstatic-secrets%2Fdev-tooling-circle%2Fledger-key
+  // akeyless path: /static-secrets/dev-tooling-circle
+  // Running `openssl ec -in key.pem -text -noout`
+  // should output the same hex pub key
   0x04,
 
-  0xb0,0x6c,0xf5,0xd8,0xf7,0xed,0x71,0xd8,
-  0xbd,0x9b,0x9d,0xc3,0x79,0x44,0xa1,0xc6,
-  0xd2,0x40,0xf6,0x9b,0xb0,0xbe,0x36,0x21,
-  0xdd,0xdb,0xb6,0xac,0xe,0xcc,0xd1,0x50,
+  0x59,0xd5,0x59,0xee,0x21,0xa7,0x76,0xfe,
+  0x43,0xe4,0xba,0xac,0x14,0x18,0x1c,0x1e,
+  0x5f,0xb5,0x1b,0x5b,0x22,0xbc,0x1,0xdd,
+  0x46,0xe0,0x60,0xe,0x8d,0xc,0xd7,0xc8,
 
-  0x8b,0xcc,0x2e,0xa4,0x62,0x27,0xe4,0x3b,
-  0x94,0x1e,0x2c,0x6f,0x1b,0x1c,0xd0,0xae,
-  0x68,0xe5,0x4b,0x18,0x5e,0x2c,0xab,0xef,
-  0x34,0x55,0x58,0x6,0x4,0xbd,0x45,0xb8
+
+  0x6d,0xc,0xf2,0x3e,0xe0,0xa3,0x8d,0xc8,
+  0x3d,0xa6,0x4b,0xd0,0x2a,0x6d,0x43,0x2c,
+  0x86,0x10,0x4a,0x47,0xaf,0xef,0x83,0x83,
+  0xc2,0x2b,0xe3,0xd4,0xd1,0xa5,0x32,0x2d
```

Comment:
Pubkey.pem
```
-----BEGIN PUBLIC KEY-----
MFYwEAYHKoZIzj0CAQYFK4EEAAoDQgAEWdVZ7iGndv5D5LqsFBgcHl+1G1sivAHd
RuBgDo0M18htDPI+4KONyD2mS9AqbUMshhBKR6/vg4PCK+PU0aUyLQ==
-----END PUBLIC KEY-----
```

Running `openssl ec -pubin -in dist/pubkey.pem -text -noout` on the key above

```
read EC key
Private-Key: (256 bit)
pub:
    04:59:d5:59:ee:21:a7:76:fe:43:e4:ba:ac:14:18:
    1c:1e:5f:b5:1b:5b:22:bc:01:dd:46:e0:60:0e:8d:
    0c:d7:c8:6d:0c:f2:3e:e0:a3:8d:c8:3d:a6:4b:d0:
    2a:6d:43:2c:86:10:4a:47:af:ef:83:83:c2:2b:e3:
    d4:d1:a5:32:2d
ASN1 OID: secp256k1
```

Which SHOULD match the C

Reply:
Data blob signed with the matching privkey:
```
AAAAaARVU0RUDio+BbyaFvUpKmFwRWpxDLicb3IAAAASAACk7DBFAiAnDzsFqjWGYzFM/38TXwwxj1MA/uL4uHc3tz1Hf9JktQIhAJwS8EREk1QvVf7Ba+Lt0USv29zvbgkDB2FcTJ2KUrBoAAAAaQRVU0RDLyXes4SMIH/I4MNANbO6f8FXYCsAAAAGAACk7DBGAiEAyYbX0QPc8OQz4INa2I7b3rZdDhSI7GjQ1Ov28QpljD4CIQCfXyjnlE2APgDbZarhS88d6kDi+3sbZVJrDK7qAzCbJQAAAGcEZVhPRnP5PcxJy4ojniAyZj6Udd1e8poIAAAAEgAApOwwRAIgccRabNzCoH70s625uSURU09dZdhKf/E1Jb/hw7zk6OACIA2DGR4k8n0I0dW4UhpK9qtU2ZW3/HQR8MuvIF54EAjrAAAAZwRjVVNEdl3oFoRYYedaJfyhIrtomLixKCoAAAASAACk7DBEAiAfghemMQt4v+Y/QOdnsUE+dIBANgNIb407LP1KqoYHRwIgV3wEzITR4K2dNExztiTg/DbGPUpfM2VKsLzYkrBo56EAAABpBGNFVVLYdjy6J2o3OObehbSzv1/e1tbKcwAAABIAAKTsMEYCIQCM1m3U0ey+jAJ0SGTXjdJwK/6qb7UwsRGM2z5/40l20wIhAM6HDu2iWlKBnRUVuWNNsKrnqVUcdhnt6+cJpOxylBXtAAAAagVjUkVBTOhTej0FbaRGZ3uenWxdtwTqq0eHAAAAEgAApOwwRgIhAMvCZRpVW+4hFAGH/oGfWV1wTToWZix/oAFsPJAEGdKfAiEA3BSz9FjkR28dgpkQWabgjO65X4CP0wqb/iXahL0LMdUAAABpBmIgY1VTRGJJKmRKWI/ZBCcL7QatUrmr/qGuAAAAEgAA83AwRAIgD2TXuM3vFW3JtK9h34BLT7D9zFYvYT44aJS6p8UoedsCIHmYoDDo7Wukyz8XlsNzEBbWBbUH6LHI3zqpBNj+fSFJAAAAagZiIGVYT0ZkwdgSZz6TvANq3D1UfZlQaW2lrwAAABIAAPNwMEUCIQCi5G3JkfojyLMN1F4805Cx1faWhiDR65ofzZdFLvHRswIgYOqsnQGWUGmXQHwwmFBzZvPaYouDUrbPDwtBV2Zb4RYAAABrB2IgY1JFQUxqDu8r7Uww3Cy0L+bF8B+A9+8W0QAAABIAAPNwMEUCIQCUrG7kT6y2FjlCBc4iJaTaUF8leam6l64fbcX0WWJmTgIgSAIwYwf6PBubwl7ny7WvK8OayCjOnmQKmJ/yIGc3vbsAAABpBmIgY0VVUvns4wEketLOIYlJQYMKJHD053TKAAAAEgAA83AwRAIgSPOsJ6UYsF4f9Bo8igrj9EjnAhnHfinGipGNmqTL57sCIAtZt0x7F+JRPHLRKxDcRRZW68mzURqPe0gZY+BNtcp7AAAAaARhIEckA9PauEPmwDs9Jx7/kXjmqWwo0l8AAAASAACu8zBFAiEA34Ldp6k6skIE4MWJNitesMIpwvZ81jgwm77sobHQhL0CIDN8lG2yMo2Msn5i+wsS8KL8UHoWOxBhTOd+eRVQyJi2AAAAaQZhIGNFVVIQyJKm7EOlPkXQuRa0t9ODsbeMDwAAABIAAK7zMEQCIC21VV1O+9q9xJw2FMo7Nb9XwNROXnVM7/JM3guvtQNhAiAQJxKZyXzJRRjyfzJxjHKwOFth9Hczg7HL+CFwB8oVuwAAAGkFYSBFRkMa17YXy2xRVqbepuR1FNFkdrmfOAAAAAYAAK7zMEUCIQD08kXowNRC43Rw4O4wkodgYiciVrYnLuZIkWCG0JhEcAIgXOn+ZqKGo8LDycjzvJOqROE5Qzh0XMwIeJkNpswFrVQAAABqBmEgVVNEQ0gi5Y3m9eSF75DfUcQc4BchMx3AAAAABgAArvMwRQIhAOl6woFCcqmCPPlMboyBPq5y7ZVIQBi+/agsk+oi8o8dAiBZqtGjVsV6RTdQynpkvv7aIQBpPC1T5vXl58B+qwmJZQAAAGkFYSBFRkN9AneQmY9xSylMlv2p4nr1htHrtQAAAAYAAK7zMEUCIQDtSpH5OVfqMnP9/l9emY9vNKT4MiR0Yw84TkxLZuYCTQIgFCF8EmAAch495NsCj3XoVOz4LJp5A/2oI/O6vMeCD1QAAABqBmEgY1VTRIdAafoesW1E1iLy4Mol7qFyNpvBAAAAEgAArvMwRQIhAJRoALAvsAj11xSkYOCH4IfHMNXd7DGCRWxUY1LhM0psAiAXTIoz0+BDoOUQ8r+MKIJtG+Xd0N/QGeyHxXNBvuJ1YQAAAGkGYSBlWE9GsPoV4AJRbQMBiEBZwKrA8McrAZ0AAAASAACu8zBEAiAhvJs0pkZyvv/NQtHJtxaPHUqKZalZgW0Aj4GzbkEO6wIgJBB7k1B/vfmvwDspbfYiQjvvJn+1VKIw6tDGIay50RAAAABqBmEgVVNEVMT4bptKWI1QHBw+JWKN/VC8jWFeAAAAEgAArvMwRQIgDWeMp/nbCoswYtk5BQ+rE/lte2o4K1m08eUm8NsVW2wCIQC/QK9ie2N8AxC3wwxByNgeJOZMl7BKxNlWHOVOzbSMhAAAAGwHYSBjUkVBTOTVF3hdCR08VIGIMttglLzCdEVFAAAAEgAArvMwRgIhAKlwXFUWBiCL4ezZ8Ybu75Zv07PWafzIDzZ8EGjBfKSrAiEA/mXa4zEjEvcepag9/FONEA0z1lEzfUffjs/Cjo/+Svk=
```

Reply:
For ease of testing on the C side, here is the parsed version of the data blob pasted above.

It's been parsed with a script from ledgerhq that can be found here: https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledgerjs/packages/hw-app-eth/src/services/ledger/erc20.ts


```ts

[
  {
    ticker: 'USDT',
    address: '0x0e2a3e05bc9a16f5292a6170456a710cb89c6f72',
    decimals: 18,
    chainId: 42220,
    signature: <Buffer 30 45 02 20 27 0f 3b 05 aa 35 86 63 31 4c ff 7f 13 5f 0c 31 8f 53 00 fe e2 f8 b8 77 37 b7 3d 47 7f d2 64 b5 02 21 00 9c 12 f0 44 44 93 54 2f 55 fe c1 ... 21 more bytes>,
    data: <Buffer 04 55 53 44 54 0e 2a 3e 05 bc 9a 16 f5 29 2a 61 70 45 6a 71 0c b8 9c 6f 72 00 00 00 12 00 00 a4 ec 30 45 02 20 27 0f 3b 05 aa 35 86 63 31 4c ff 7f 13 ... 54 more bytes>,
    dataRaw: 'BFVTRFQOKj4FvJoW9SkqYXBFanEMuJxvcgAAABIAAKTsMEUCICcPOwWqNYZjMUz/fxNfDDGPUwD+4vi4dze3PUd/0mS1AiEAnBLwRESTVC9V/sFr4u3RRK/b3O9uCQMHYVxMnYpSsGg='
  },
  {
    ticker: 'USDC',
    address: '0x2f25deb3848c207fc8e0c34035b3ba7fc157602b',
    decimals: 6,
    chainId: 42220,
    signature: <Buffer 30 46 02 21 00 c9 86 d7 d1 03 dc f0 e4 33 e0 83 5a d8 8e db de b6 5d 0e 14 88 ec 68 d0 d4 eb f6 f1 0a 65 8c 3e 02 21 00 9f 5f 28 e7 94 4d 80 3e 00 db ... 22 more bytes>,
    data: <Buffer 04 55 53 44 43 2f 25 de b3 84 8c 20 7f c8 e0 c3 40 35 b3 ba 7f c1 57 60 2b 00 00 00 06 00 00 a4 ec 30 46 02 21 00 c9 86 d7 d1 03 dc f0 e4 33 e0 83 5a ... 55 more bytes>,
    dataRaw: 'BFVTREMvJd6zhIwgf8jgw0A1s7p/wVdgKwAAAAYAAKTsMEYCIQDJhtfRA9zw5DPgg1rYjtvetl0OFIjsaNDU6/bxCmWMPgIhAJ9fKOeUTYA+ANtlquFLzx3qQOL7extlUmsMruoDMJsl'
  },
  {
    ticker: 'eXOF',
    address: '0x73f93dcc49cb8a239e2032663e9475dd5ef29a08',
    decimals: 18,
    chainId: 42220,
    signature: <Buffer 30 44 02 20 71 c4 5a 6c dc c2 a0 7e f4 b3 ad b9 b9 25 11 53 4f 5d 65 d8 4a 7f f1 35 25 bf e1 c3 bc e4 e8 e0 02 20 0d 83 19 1e 24 f2 7d 08 d1 d5 b8 52 ... 20 more bytes>,
    data: <Buffer 04 65 58 4f 46 73 f9 3d cc 49 cb 8a 23 9e 20 32 66 3e 94 75 dd 5e f2 9a 08 00 00 00 12 00 00 a4 ec 30 44 02 20 71 c4 5a 6c dc c2 a0 7e f4 b3 ad b9 b9 ... 53 more bytes>,
    dataRaw: 'BGVYT0Zz+T3MScuKI54gMmY+lHXdXvKaCAAAABIAAKTsMEQCIHHEWmzcwqB+9LOtubklEVNPXWXYSn/xNSW/4cO85OjgAiANgxkeJPJ9CNHVuFIaSvarVNmVt/x0EfDLryBeeBAI6w=='
  },
  {
    ticker: 'cUSD',
    address: '0x765de816845861e75a25fca122bb6898b8b1282a',
    decimals: 18,
    chainId: 42220,
    signature: <Buffer 30 44 02 20 1f 82 17 a6 31 0b 78 bf e6 3f 40 e7 67 b1 41 3e 74 80 40 36 03 48 6f 8d 3b 2c fd 4a aa 86 07 47 02 20 57 7c 04 cc 84 d1 e0 ad 9d 34 4c 73 ... 20 more bytes>,
    data: <Buffer 04 63 55 53 44 76 5d e8 16 84 58 61 e7 5a 25 fc a1 22 bb 68 98 b8 b1 28 2a 00 00 00 12 00 00 a4 ec 30 44 02 20 1f 82 17 a6 31 0b 78 bf e6 3f 40 e7 67 ... 53 more bytes>,
    dataRaw: 'BGNVU0R2XegWhFhh51ol/KEiu2iYuLEoKgAAABIAAKTsMEQCIB+CF6YxC3i/5j9A52exQT50gEA2A0hvjTss/UqqhgdHAiBXfATMhNHgrZ00THO2JOD8NsY9Sl8zZUqwvNiSsGjnoQ=='
  },
  {
    ticker: 'cEUR',
    address: '0xd8763cba276a3738e6de85b4b3bf5fded6d6ca73',
    decimals: 18,
    chainId: 42220,
    signature: <Buffer 30 46 02 21 00 8c d6 6d d4 d1 ec be 8c 02 74 48 64 d7 8d d2 70 2b fe aa 6f b5 30 b1 11 8c db 3e 7f e3 49 76 d3 02 21 00 ce 87 0e ed a2 5a 52 81 9d 15 ... 22 more bytes>,
    data: <Buffer 04 63 45 55 52 d8 76 3c ba 27 6a 37 38 e6 de 85 b4 b3 bf 5f de d6 d6 ca 73 00 00 00 12 00 00 a4 ec 30 46 02 21 00 8c d6 6d d4 d1 ec be 8c 02 74 48 64 ... 55 more bytes>,
    dataRaw: 'BGNFVVLYdjy6J2o3OObehbSzv1/e1tbKcwAAABIAAKTsMEYCIQCM1m3U0ey+jAJ0SGTXjdJwK/6qb7UwsRGM2z5/40l20wIhAM6HDu2iWlKBnRUVuWNNsKrnqVUcdhnt6+cJpOxylBXt'
  },
  {
    ticker: 'cREAL',
    address: '0xe8537a3d056da446677b9e9d6c5db704eaab4787',
    decimals: 18,
    chainId: 42220,
    signature: <Buffer 30 46 02 21 00 cb c2 65 1a 55 5b ee 21 14 01 87 fe 81 9f 59 5d 70 4d 3a 16 66 2c 7f a0 01 6c 3c 90 04 19 d2 9f 02 21 00 dc 14 b3 f4 58 e4 47 6f 1d 82 ... 22 more bytes>,
    data: <Buffer 05 63 52 45 41 4c e8 53 7a 3d 05 6d a4 46 67 7b 9e 9d 6c 5d b7 04 ea ab 47 87 00 00 00 12 00 00 a4 ec 30 46 02 21 00 cb c2 65 1a 55 5b ee 21 14 01 87 ... 56 more bytes>,
    dataRaw: 'BWNSRUFM6FN6PQVtpEZne56dbF23BOqrR4cAAAASAACk7DBGAiEAy8JlGlVb7iEUAYf+gZ9ZXXBNOhZmLH+gAWw8kAQZ0p8CIQDcFLP0WORHbx2CmRBZpuCM7rlfgI/TCpv+JdqEvQsx1Q=='
  },
  {
    ticker: 'b cUSD',
    address: '0x62492a644a588fd904270bed06ad52b9abfea1ae',
    decimals: 18,
    chainId: 62320,
    signature: <Buffer 30 44 02 20 0f 64 d7 b8 cd ef 15 6d c9 b4 af 61 df 80 4b 4f b0 fd cc 56 2f 61 3e 38 68 94 ba a7 c5 28 79 db 02 20 79 98 a0 30 e8 ed 6b a4 cb 3f 17 96 ... 20 more bytes>,
    data: <Buffer 06 62 20 63 55 53 44 62 49 2a 64 4a 58 8f d9 04 27 0b ed 06 ad 52 b9 ab fe a1 ae 00 00 00 12 00 00 f3 70 30 44 02 20 0f 64 d7 b8 cd ef 15 6d c9 b4 af ... 55 more bytes>,
    dataRaw: 'BmIgY1VTRGJJKmRKWI/ZBCcL7QatUrmr/qGuAAAAEgAA83AwRAIgD2TXuM3vFW3JtK9h34BLT7D9zFYvYT44aJS6p8UoedsCIHmYoDDo7Wukyz8XlsNzEBbWBbUH6LHI3zqpBNj+fSFJ'
  },
  {
    ticker: 'b eXOF',
    address: '0x64c1d812673e93bc036adc3d547d9950696da5af',
    decimals: 18,
    chainId: 62320,
    signature: <Buffer 30 45 02 21 00 a2 e4 6d c9 91 fa 23 c8 b3 0d d4 5e 3c d3 90 b1 d5 f6 96 86 20 d1 eb 9a 1f cd 97 45 2e f1 d1 b3 02 20 60 ea ac 9d 01 96 50 69 97 40 7c ... 21 more bytes>,
    data: <Buffer 06 62 20 65 58 4f 46 64 c1 d8 12 67 3e 93 bc 03 6a dc 3d 54 7d 99 50 69 6d a5 af 00 00 00 12 00 00 f3 70 30 45 02 21 00 a2 e4 6d c9 91 fa 23 c8 b3 0d ... 56 more bytes>,
    dataRaw: 'BmIgZVhPRmTB2BJnPpO8A2rcPVR9mVBpbaWvAAAAEgAA83AwRQIhAKLkbcmR+iPIsw3UXjzTkLHV9paGINHrmh/Nl0Uu8dGzAiBg6qydAZZQaZdAfDCYUHNm89pii4NSts8PC0FXZlvhFg=='
  },
  {
    ticker: 'b cREAL',
    address: '0x6a0eef2bed4c30dc2cb42fe6c5f01f80f7ef16d1',
    decimals: 18,
    chainId: 62320,
    signature: <Buffer 30 45 02 21 00 94 ac 6e e4 4f ac b6 16 39 42 05 ce 22 25 a4 da 50 5f 25 79 a9 ba 97 ae 1f 6d c5 f4 59 62 66 4e 02 20 48 02 30 63 07 fa 3c 1b 9b c2 5e ... 21 more bytes>,
    data: <Buffer 07 62 20 63 52 45 41 4c 6a 0e ef 2b ed 4c 30 dc 2c b4 2f e6 c5 f0 1f 80 f7 ef 16 d1 00 00 00 12 00 00 f3 70 30 45 02 21 00 94 ac 6e e4 4f ac b6 16 39 ... 57 more bytes>,
    dataRaw: 'B2IgY1JFQUxqDu8r7Uww3Cy0L+bF8B+A9+8W0QAAABIAAPNwMEUCIQCUrG7kT6y2FjlCBc4iJaTaUF8leam6l64fbcX0WWJmTgIgSAIwYwf6PBubwl7ny7WvK8OayCjOnmQKmJ/yIGc3vbs='
  },
  {
    ticker: 'b cEUR',
    address: '0xf9ece301247ad2ce21894941830a2470f4e774ca',
    decimals: 18,
    chainId: 62320,
    signature: <Buffer 30 44 02 20 48 f3 ac 27 a5 18 b0 5e 1f f4 1a 3c 8a 0a e3 f4 48 e7 02 19 c7 7e 29 c6 8a 91 8d 9a a4 cb e7 bb 02 20 0b 59 b7 4c 7b 17 e2 51 3c 72 d1 2b ... 20 more bytes>,
    data: <Buffer 06 62 20 63 45 55 52 f9 ec e3 01 24 7a d2 ce 21 89 49 41 83 0a 24 70 f4 e7 74 ca 00 00 00 12 00 00 f3 70 30 44 02 20 48 f3 ac 27 a5 18 b0 5e 1f f4 1a ... 55 more bytes>,
    dataRaw: 'BmIgY0VVUvns4wEketLOIYlJQYMKJHD053TKAAAAEgAA83AwRAIgSPOsJ6UYsF4f9Bo8igrj9EjnAhnHfinGipGNmqTL57sCIAtZt0x7F+JRPHLRKxDcRRZW68mzURqPe0gZY+BNtcp7'
  },
  {
    ticker: 'a G$',
    address: '0x03d3dab843e6c03b3d271eff9178e6a96c28d25f',
    decimals: 18,
    chainId: 44787,
    signature: <Buffer 30 45 02 21 00 df 82 dd a7 a9 3a b2 42 04 e0 c5 89 36 2b 5e b0 c2 29 c2 f6 7c d6 38 30 9b be ec a1 b1 d0 84 bd 02 20 33 7c 94 6d b2 32 8d 8c b2 7e 62 ... 21 more bytes>,
    data: <Buffer 04 61 20 47 24 03 d3 da b8 43 e6 c0 3b 3d 27 1e ff 91 78 e6 a9 6c 28 d2 5f 00 00 00 12 00 00 ae f3 30 45 02 21 00 df 82 dd a7 a9 3a b2 42 04 e0 c5 89 ... 54 more bytes>,
    dataRaw: 'BGEgRyQD09q4Q+bAOz0nHv+ReOapbCjSXwAAABIAAK7zMEUCIQDfgt2nqTqyQgTgxYk2K16wwinC9nzWODCbvuyhsdCEvQIgM3yUbbIyjYyyfmL7CxLwovxQehY7EGFM5355FVDImLY='
  },
  {
    ticker: 'a cEUR',
    address: '0x10c892a6ec43a53e45d0b916b4b7d383b1b78c0f',
    decimals: 18,
    chainId: 44787,
    signature: <Buffer 30 44 02 20 2d b5 55 5d 4e fb da bd c4 9c 36 14 ca 3b 35 bf 57 c0 d4 4e 5e 75 4c ef f2 4c de 0b af b5 03 61 02 20 10 27 12 99 c9 7c c9 45 18 f2 7f 32 ... 20 more bytes>,
    data: <Buffer 06 61 20 63 45 55 52 10 c8 92 a6 ec 43 a5 3e 45 d0 b9 16 b4 b7 d3 83 b1 b7 8c 0f 00 00 00 12 00 00 ae f3 30 44 02 20 2d b5 55 5d 4e fb da bd c4 9c 36 ... 55 more bytes>,
    dataRaw: 'BmEgY0VVUhDIkqbsQ6U+RdC5FrS304Oxt4wPAAAAEgAArvMwRAIgLbVVXU772r3EnDYUyjs1v1fA1E5edUzv8kzeC6+1A2ECIBAnEpnJfMlFGPJ/MnGMcrA4W2H0dzODscv4IXAHyhW7'
  },
  {
    ticker: 'a EFC',
    address: '0x1ad7b617cb6c5156a6dea6e47514d16476b99f38',
    decimals: 6,
    chainId: 44787,
    signature: <Buffer 30 45 02 21 00 f4 f2 45 e8 c0 d4 42 e3 74 70 e0 ee 30 92 87 60 62 27 22 56 b6 27 2e e6 48 91 60 86 d0 98 44 70 02 20 5c e9 fe 66 a2 86 a3 c2 c3 c9 c8 ... 21 more bytes>,
    data: <Buffer 05 61 20 45 46 43 1a d7 b6 17 cb 6c 51 56 a6 de a6 e4 75 14 d1 64 76 b9 9f 38 00 00 00 06 00 00 ae f3 30 45 02 21 00 f4 f2 45 e8 c0 d4 42 e3 74 70 e0 ... 55 more bytes>,
    dataRaw: 'BWEgRUZDGte2F8tsUVam3qbkdRTRZHa5nzgAAAAGAACu8zBFAiEA9PJF6MDUQuN0cODuMJKHYGInIla2Jy7mSJFghtCYRHACIFzp/maihqPCw8nI87yTqkThOUM4dFzMCHiZDabMBa1U'
  },
  {
    ticker: 'a USDC',
    address: '0x4822e58de6f5e485ef90df51c41ce01721331dc0',
    decimals: 6,
    chainId: 44787,
    signature: <Buffer 30 45 02 21 00 e9 7a c2 81 42 72 a9 82 3c f9 4c 6e 8c 81 3e ae 72 ed 95 48 40 18 be fd a8 2c 93 ea 22 f2 8f 1d 02 20 59 aa d1 a3 56 c5 7a 45 37 50 ca ... 21 more bytes>,
    data: <Buffer 06 61 20 55 53 44 43 48 22 e5 8d e6 f5 e4 85 ef 90 df 51 c4 1c e0 17 21 33 1d c0 00 00 00 06 00 00 ae f3 30 45 02 21 00 e9 7a c2 81 42 72 a9 82 3c f9 ... 56 more bytes>,
    dataRaw: 'BmEgVVNEQ0gi5Y3m9eSF75DfUcQc4BchMx3AAAAABgAArvMwRQIhAOl6woFCcqmCPPlMboyBPq5y7ZVIQBi+/agsk+oi8o8dAiBZqtGjVsV6RTdQynpkvv7aIQBpPC1T5vXl58B+qwmJZQ=='
  },
  {
    ticker: 'a EFC',
    address: '0x7d027790998f714b294c96fda9e27af586d1ebb5',
    decimals: 6,
    chainId: 44787,
    signature: <Buffer 30 45 02 21 00 ed 4a 91 f9 39 57 ea 32 73 fd fe 5f 5e 99 8f 6f 34 a4 f8 32 24 74 63 0f 38 4e 4c 4b 66 e6 02 4d 02 20 14 21 7c 12 60 00 72 1e 3d e4 db ... 21 more bytes>,
    data: <Buffer 05 61 20 45 46 43 7d 02 77 90 99 8f 71 4b 29 4c 96 fd a9 e2 7a f5 86 d1 eb b5 00 00 00 06 00 00 ae f3 30 45 02 21 00 ed 4a 91 f9 39 57 ea 32 73 fd fe ... 55 more bytes>,
    dataRaw: 'BWEgRUZDfQJ3kJmPcUspTJb9qeJ69YbR67UAAAAGAACu8zBFAiEA7UqR+TlX6jJz/f5fXpmPbzSk+DIkdGMPOE5MS2bmAk0CIBQhfBJgAHIePeTbAo916FTs+CyaeQP9qCPzurzHgg9U'
  },
  {
    ticker: 'a cUSD',
    address: '0x874069fa1eb16d44d622f2e0ca25eea172369bc1',
    decimals: 18,
    chainId: 44787,
    signature: <Buffer 30 45 02 21 00 94 68 00 b0 2f b0 08 f5 d7 14 a4 60 e0 87 e0 87 c7 30 d5 dd ec 31 82 45 6c 54 63 52 e1 33 4a 6c 02 20 17 4c 8a 33 d3 e0 43 a0 e5 10 f2 ... 21 more bytes>,
    data: <Buffer 06 61 20 63 55 53 44 87 40 69 fa 1e b1 6d 44 d6 22 f2 e0 ca 25 ee a1 72 36 9b c1 00 00 00 12 00 00 ae f3 30 45 02 21 00 94 68 00 b0 2f b0 08 f5 d7 14 ... 56 more bytes>,
    dataRaw: 'BmEgY1VTRIdAafoesW1E1iLy4Mol7qFyNpvBAAAAEgAArvMwRQIhAJRoALAvsAj11xSkYOCH4IfHMNXd7DGCRWxUY1LhM0psAiAXTIoz0+BDoOUQ8r+MKIJtG+Xd0N/QGeyHxXNBvuJ1YQ=='
  },
  {
    ticker: 'a eXOF',
    address: '0xb0fa15e002516d0301884059c0aac0f0c72b019d',
    decimals: 18,
    chainId: 44787,
    signature: <Buffer 30 44 02 20 21 bc 9b 34 a6 46 72 be ff cd 42 d1 c9 b7 16 8f 1d 4a 8a 65 a9 59 81 6d 00 8f 81 b3 6e 41 0e eb 02 20 24 10 7b 93 50 7f bd f9 af c0 3b 29 ... 20 more bytes>,
    data: <Buffer 06 61 20 65 58 4f 46 b0 fa 15 e0 02 51 6d 03 01 88 40 59 c0 aa c0 f0 c7 2b 01 9d 00 00 00 12 00 00 ae f3 30 44 02 20 21 bc 9b 34 a6 46 72 be ff cd 42 ... 55 more bytes>,
    dataRaw: 'BmEgZVhPRrD6FeACUW0DAYhAWcCqwPDHKwGdAAAAEgAArvMwRAIgIbybNKZGcr7/zULRybcWjx1KimWpWYFtAI+Bs25BDusCICQQe5NQf735r8A7KW32IkI77yZ/tVSiMOrQxiGsudEQ'
  },
  {
    ticker: 'a USDT',
    address: '0xc4f86e9b4a588d501c1c3e25628dfd50bc8d615e',
    decimals: 18,
    chainId: 44787,
    signature: <Buffer 30 45 02 20 0d 67 8c a7 f9 db 0a 8b 30 62 d9 39 05 0f ab 13 f9 6d 7b 6a 38 2b 59 b4 f1 e5 26 f0 db 15 5b 6c 02 21 00 bf 40 af 62 7b 63 7c 03 10 b7 c3 ... 21 more bytes>,
    data: <Buffer 06 61 20 55 53 44 54 c4 f8 6e 9b 4a 58 8d 50 1c 1c 3e 25 62 8d fd 50 bc 8d 61 5e 00 00 00 12 00 00 ae f3 30 45 02 20 0d 67 8c a7 f9 db 0a 8b 30 62 d9 ... 56 more bytes>,
    dataRaw: 'BmEgVVNEVMT4bptKWI1QHBw+JWKN/VC8jWFeAAAAEgAArvMwRQIgDWeMp/nbCoswYtk5BQ+rE/lte2o4K1m08eUm8NsVW2wCIQC/QK9ie2N8AxC3wwxByNgeJOZMl7BKxNlWHOVOzbSMhA=='
  },
  {
    ticker: 'a cREAL',
    address: '0xe4d517785d091d3c54818832db6094bcc2744545',
    decimals: 18,
    chainId: 44787,
    signature: <Buffer 30 46 02 21 00 a9 70 5c 55 16 06 20 8b e1 ec d9 f1 86 ee ef 96 6f d3 b3 d6 69 fc c8 0f 36 7c 10 68 c1 7c a4 ab 02 21 00 fe 65 da e3 31 23 12 f7 1e a5 ... 22 more bytes>,
    data: <Buffer 07 61 20 63 52 45 41 4c e4 d5 17 78 5d 09 1d 3c 54 81 88 32 db 60 94 bc c2 74 45 45 00 00 00 12 00 00 ae f3 30 46 02 21 00 a9 70 5c 55 16 06 20 8b e1 ... 58 more bytes>,
    dataRaw: 'B2EgY1JFQUzk1Rd4XQkdPFSBiDLbYJS8wnRFRQAAABIAAK7zMEYCIQCpcFxVFgYgi+Hs2fGG7u+Wb9Oz1mn8yA82fBBowXykqwIhAP5l2uMxIxL3HqWoPfxTjRANM9ZRM31H347Pwo6P/kr5'
  }
]
```